go/vt/vtctl/reparentutil/replication.go

```diff
@@ -222,6 +226,7 @@ func StopReplicationAndBuildStatusMaps(
 		tabletAliasToWaitFor = topoproto.TabletAliasString(tabletToWaitFor)
 	}
 	for alias, tabletInfo := range tabletMap {
+		allTablets = append(allTablets, tabletInfo.Tablet)
 		if !ignoredTablets.Has(alias) {
```

Comment:
How does this work with `ignoredTablets`? If we allow 1 failure but `ignoredTablets` is not empty, what happens?

Reply:
IgnoredTablets functionality remains the same. We used to proceed with 1 failure in non-ignored tablets earlier which we will continue to do.
For the revocation check, we use the entire list of tablets to check for revocation to be sure that we only proceed with multiple failures if we can guarantee safety.